cardigan/stories/components/DescriptionList/DescriptionList.stories.tsx

```diff
@@ -0,0 +1,27 @@
+import DescriptionList from '@weco/common/views/components/DescriptionList/DescriptionList';
+
+const Template = args => <DescriptionList {...args} />;
+export const basic = Template.bind({});
+basic.args = {
```

Comment:
This is new - what are we doing here?

Reply:
This is Storybook 6's way to do what was previously handled by the knobs addon. https://storybook.js.org/docs/riot/writing-stories/args